Gemfile

```diff
@@ -17,6 +17,10 @@ gem 'jbuilder', '~> 2.5'
 gem 'geocoder'
 gem 'dotenv-rails'
 gem 'rails-assets-gmaps', source: 'https://rails-assets.org'
+gem 'jquery-rails'
+gem 'bootstrap', '~> 4.0'
+# gem 'sprockets-rails', '~> 3.2', '>= 3.2.1'
```

Comment:
Remove if not used...

Reply:
Thanks, didn't see that one. Will remove 👍